gnovm/stdlibs/std/coins.gno

```diff
@@ -177,6 +177,15 @@ func (cz Coins) Add(b Coins) Coins {
 	return c
 }
 
+func (cz Coins) GetCoin(denom string) Coin {
```

Comment:
I would say something more like `ForDenom`.

I would also suggest to return `Coin{Denom: denom}` instead of the empty value, so that if you get `cz.ForDenom("ugnot").String()`, and you have 0, the output is `0ugnot`.

cc/ @leohhhn on the naming